packages/angular/testing/src/testing-utils.ts

```diff
@@ -2,6 +2,8 @@ import type { Observable } from 'rxjs';
 import { first, toArray } from 'rxjs/operators';
 
 /**
+ * @deprecated This will be removed in a later version of Nx. Since RxJS 7, use firstValueFrom(obs$.pipe(toArray())).
```

Comment:
Let's suggest both `firstValueFrom` and `lastValueFrom`. Both are valid options and by suggesting only one of them, we might mislead folks to think is "the" way.